docs/treefile.md

```diff
@@ -138,6 +138,10 @@ It supports the following parameters:
     to the UNIX epoch time to be used as the build timestamp. Currently only
     fully supports the `bdb` backend. Somewhat experimental.
 
+ * `label-usretc-as-etc`: boolean, optional: If enabled, this sets the SELinux label for
```

Comment:
I wonder if we even need a new knob for this. Should be safe to always do this, right? Since we basically own `/usr/etc` anyway.

Reply:
Yes, I'm being conservative here by making it opt in (both here and on the ostree side).

Dunno.  I guess we could try to actually do it by default ostree side, and add an option to *disable* it in case we regress things somehow.

Reply:
This is a nontrivial change though, and so far we've been really conservative with options that *should* be on by default but aren't (sysroot.readonly is a good example) to ensure that updating ostree doesn't break anything.

Reply:
Yeah, I'm less sure about making it default in libostree. I think it makes sense to make it opt in there. For rpm-ostree, it seems safer to do but cool of course to be conservative there.

Reply:
I think making it opt-in in ostree. but always enable in rpm-ostree (possibly with an opt-out) is the right approach.

Reply:
It's painful every time we add an opt-in for sure because we end up having to update so many different manifest/treefiles (coreos vs edge vs atomic desktop etc etc.)

However I still just feel we need to be conservative because this can easily have non-obvious effects.

Even on-by-default with a knob to turn it off gets tricky because then the build system requires a *new* rpm-ostree (old ones will barf on the unexpected field).

Reply:
Not only that, we need a new version of osbuild, and any other tools that call out to rpm-ostree to compose a commit.